source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md

```diff
@@ -17,6 +17,30 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
 
 <div class="toc-list"></div>
 
+## Shopware 5.3 RC 2
+
+### Smarty security mode
+We have activated the Smarty security mode globally with 5.3:
+https://github.com/shopware/shopware/blob/5.3/engine/Shopware/Components/DependencyInjection/Bridge/Template.php#L56
```

Comment:
You should use at revisioned URL here like https://github.com/shopware/shopware/blob/e63da07761b00cb76b02e8ea3aca9b88817a3acd/engine/Shopware/Components/DependencyInjection/Bridge/Template.php#L56.

Else the Filename or line may move with later commits.
See: https://help.github.com/articles/getting-permanent-links-to-files/